driver/src/clawfc/CLAWIgnoreLineRecognizer.g4

```diff
@@ -0,0 +1,51 @@
+/*
+ * This file is released under terms of BSD license
+ * See LICENSE file for more information
+ * @author Mikhail Zhigun
+*/
+/**
+ * ANTLR 4 Grammar file for parsing line with applied CLAW Ignore directive.
+ */
+grammar CLAWIgnoreLineRecognizer;
+
+root : contents EOF;
+
+contents: CONTENTS;
+
+CLAW_IGNORE_DIR_START: SEP? '!' SEP? '$' CLAW SEP IGNORE SEP ->skip;
+CONTENTS : '|' NOT_EOL_CHR* '|';
+CLAW_IGNORE_DIR_END : SEP? ('\r')? '\n' ->skip;
+
+fragment NOT_EOL_CHR:  ~[\r\n];
+fragment CLAW : C L A W;
```

Comment:
Duplicated and overkill. Same for the rest of the file.

Reply:
Makes maintenance easier